code/modules/tgui/tgui_window.dm

```diff
@@ -193,7 +193,7 @@
  *
  * optional can_be_suspended bool
  */
-/datum/tgui_window/proc/close(can_be_suspended = TRUE, logout = FALSE)
```

Comment:
and yes i just updated the entire file to tg's sue me

Reply:
Literally the correct way to go about it